src/components/AppElements/components/CodeEditor/codeGenerators/htmlImportCodeGenerator.js

```diff
@@ -1,22 +1,61 @@
-export default function () {
-	return `const htmlString = document.querySelector('textarea').value;
+export default function (importConfig) {
```

Comment:
Everywhere where it's "  ".repeat() change it to "\t".repeat().

See the editor one for example